demisto_sdk/commands/upload/tests/uploader_test.py

```diff
@@ -897,10 +897,8 @@ def test_upload_without_skip_validate(self, mocker, path: Path, version: str):
                     "metadata.json",
                     "pack_metadata.json",
                     "XSIAMDashboards/",
-                    "XSIAMDashboards/xsiamdashboard-MyDashboard.json",
                     "XSIAMDashboards/external-xsiamdashboard-MyDashboard.json",
                     "Triggers/",
-                    "Triggers/trigger-MyTrigger.json",
```

Comment:
why are these removed?

Reply:
Setting the default fromversion of this items to 8.0.0 prevents uploads only the external version of the files.